ref/javascript/segwit_addr.js

```diff
@@ -1,163 +1,71 @@
-bech32 = function() {
-    var BECH32_CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l";
-    var BECH32_GENERATOR = [0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3];
+var bech32 = require('./bech32');
```

Comment:
with browserify, which is what everyone uses, yes.

You could also put in a precompiled dist.js that outputs an AMD compatible export, but that's so-5-years-ago.